Package errors (#8529)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/packages.md

    warn-unpinned: false
```

## Troubleshooting

> If you encounter errors while working with dbt packages, see the following troubleshooting FAQ pages:

Reviewer: If we're using FAQ paths, we can change this to
```suggestion
If you encounter errors while working with dbt packages, see the following FAQs:
```

> - [Why am I receiving a Runtime Error in my packages?](/faqs/Troubleshooting/runtime-packages.yml)

Reviewer: I suggest we use FAQ paths for these so that users don't have to open a different page

> - [[Error] Could not find my_project package”](/faqs/Troubleshooting/dispatch-could-not-find-package)

Author: ```suggestion
- [[Error] Could not find my_project package](/faqs/Troubleshooting/dispatch-could-not-find-package)
```

> </File>

Reviewer: Move closing `</File>` before line 463

```

</File>

## Troubleshooting

If you encounter errors while working with dbt packages, see the following FAQs:

> - <FAQ path="Troubleshooting/runtime-packages.yml" />

Reviewer: ```suggestion
<FAQ path="Troubleshooting/runtime-packages.yml" />
```

> - <FAQ path="Troubleshooting/dispatch-could-not-find-package" />

Reviewer: ```suggestion
<FAQ path="Troubleshooting/dispatch-could-not-find-package" />
```
